lib/keychain/keychain.rb

```diff
@@ -1,5 +1,33 @@
 
 module Sec
+  enum :SecItemImportExportFlags, [:kSecItemPemArmour, 1]
+
+  enum :SecExternalFormat, [:kSecFormatUnknown, 0,
```

Comment:
Some of these constants are already defined in key.rb - could you consolidate so that we are only attaching once?